### PR TITLE
chore(yoroi): update readme

### PR DESCRIPTION
**Pull request by banklesss**
Updating readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,58 @@
...
+
+## Features
+
+- **Lightweight**: Yoroi is a lightweight wallet that does not require downloading the full blockchain, making it quick to set up and use.
+- **Secure**: Your private keys are encrypted and never shared with our servers or third parties.
+- **Fast Transactions**: Easily send and receive ADA with fast transaction speeds.
+- **Cross-Platform**: Available as a mobile app for iOS and Android.
+- **Multiple Wallets**: Manage multiple wallets with ease.
+- **Hardware Wallet Support**: Compatible with Ledger and Trezor hardware wallets for added security.
+- **Open Source**: Fully open-source software under the MIT License.
```
Comment (michalszorad) [inline README.md]: MIT or Apache License?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,58 @@
...
+2. **Restore an Existing Wallet**: If you have an existing wallet, choose "Restore Wallet" and enter your recovery phrase.
+3. **Send and Receive ADA**: Use the "Send" and "Receive" tabs to manage your ADA transactions.
+4. **Connect Hardware Wallet**: To use a hardware wallet, select "Connect Hardware Wallet" and follow the prompts.
+5. **Swap**: Instantly exchange ADA for other cryptocurrencies directly within the Yoroi wallet.
+6. **Buy/Sell ADA**: Easily purchase or sell ADA directly through the Yoroi wallet using integrated partner services.
+7. **Transaction History and Data Review**: View detailed transaction history and data directly in the Yoroi wallet.
+8. **Portfolio Overview**: Easily review and manage information about your ADA and other tokens with a comprehensive portfolio dashboard.
+9. **Catalyst Voting**: Participate in Project Catalyst by voting on Cardano governance proposals directly from the Yoroi wallet.
+10. **Stake to Pools**: Delegate your ADA to staking pools directly from Yoroi to earn rewards while supporting the network's security and decentralization.
+
```
Comment (michalszorad) [inline README.md]: Maybe worth mentioning dapp connector as well?

Comment (banklesss) [reply README.md]: yeah !!!

Comment (banklesss) [reply README.md]: governance is not live yet so i didnt added it

**Review by michalszorad — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (banklesss) [reply README.md]: fixed

**Comment by rahulnr7** [comment]
@banklesss please consider the following :
1)  governance, 
2) proof of onboarding (CIP 99 - scan me), 
3) domain support for CNS, adahandle & unstoppable

**Comment by banklesss** [comment]
updated @rahulnr7

**Pull request merged**